docs/docs/_data/analysis/sidebar.yml

```diff
@@ -14,6 +14,9 @@ options:
     - title: Types and invariants
       url: /analysis/types
 
+    - title: Wrappers
```

Comment:
Could it make sense to call this `Types` or `Classes`?
Since we also have "Fields". "Wrappers" is often related to a negative connotatiion.